logging/nox.py

```diff
@@ -73,7 +73,7 @@ def lint(session):
     Returns a failure if flake8 finds linting errors or sufficiently
     serious code quality issues.
     """
-    session.interpreter = 'python3.6'
+    session.interpreter = 'python3.5'
```

Comment:
Yes sorry didn't mean to include this. Yeah was just being lazy cause don't have 3.6 installed.